browser/ui/webui/new_tab_page/BUILD.gn

```diff
@@ -7,10 +7,7 @@ source_set("unit_tests") {
   testonly = true
 
   if (!is_android) {
```

Comment:
why is this !is_android?

Reply:
> why is this !is_android?

@simonhong @petemill Could you tell us why?

Reply:
Because these tests aren't relevant in android since the webui version of the New Tab Page isn't used on android? Is it better to wrap the unittest dep on `browser/ui/webui/new_tab_page:unit_tests` with `if (!is_android)` at a higher level @bridiver - is that what you're referencing?